src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1218,6 +1221,36 @@ public static string GetDecoratedSymbolName(SymbolReference symbolReference)
             await requestContext.SendResultAsync(result);
         }
 
+        protected async Task HandleGetRunspaceRequestAsync(
+            object noParams,
+            RequestContext<GetRunspaceResponse[]> requestContext)
+        {
+            var runspaceResponses = new List<GetRunspaceResponse>();
+
+            if (this.editorSession.PowerShellContext.LocalPowerShellVersion.Version.Major >= 5)
```

Comment:
Was `Get-Runspace` added in 5?

Reply:
That's a really good question. I just took that check from the process picker but I'm not sure when Debug-Runspace\Get-Runspace was added. I can try and find out.

Reply:
![image](https://user-images.githubusercontent.com/1256531/54156425-d4fa1900-440b-11e9-8a0d-f22cb72bbc8d.png)

Reply:
Well looks like that answers our question :)